spec/features/submit_feedback_spec.rb

```diff
@@ -1,26 +1,58 @@
 require 'rails_helper'
 
 RSpec.feature 'Submit feedback', js: true do
-  before do skip 'Features specs not yet fixed' end
-
   include FeaturesHelper
 
-  scenario 'submitting feedback' do
+  normalised_body = lambda do |r1, r2|
```

Comment:
Cool, I think it would be good to add this as a comment so is clear to others in the future why this is needed